anitya/templates/project_new.html

```diff
@@ -45,8 +45,14 @@ <h1>{{ context }} project</h1>
             <tr id="version_filter_row">
                 {{ render_field_in_row(form.version_filter, tabindex=9) }}
             </tr>
+            <tr id="cpe_2_3_row">
```

Comment:
Adding new fields here needs to be reflected in `anitya.ui.new_project`, `anitya.forms.ProjectForm` and `anitya.api_v2.ProjectsResource.post`.